### ISSUE TITLE
Export to opml format

### ISSUE THREAD

**Issue by ghost** [SkyTubeTeam/SkyTube#204]
Hi all,
i want to export my subscriptions to opml format ,or the skytube format is not a opml format how to do it ?

thanks for your help.

**Comment by ram-on** [comment]
Currently you can only export to SkyTube format.  Is there a reason why you want to export to OPML format?

**Comment by emporeso2** [comment]
Yes , i want to check my feeds in my rss reader and then watch them via skytube ?

**Comment by kototama** [comment]
This issue is important to avoid a lock-in of the users. I had to use [this shell script](https://github.com/T-vK/subscriptions-extractor) to export the feeds I accumulated over the years!
